data/human/culture conversations.txt

```diff
@@ -791,3 +791,13 @@ mission "Raider returning to Mordente-Bridi"
 			`Watching the microgravity section of the spaceport from one of the bars in Mordente Cylinder, you observe a Bastion in pirate markings coming nearly too fast for safe approach to the station, its hull covered in gashes and burns from what looks to be Plasma Cannon shots.`
 			`	As it comes to a stop in a maintenance bay, the spaceport's engineering shuttles appear from all over and begin buzzing around the Bastion, assessing the damage and beginning the work on repairs. Meanwhile, its crew disperse from the ship and immediately begin unloading a multitude of busted-up crates and containers from the hold, probably raided from a civilian transport. It looks like their haul was worth the damage.`
 			`	Within the hour the work shuttles scatter from the ship as they finish patching up its battered hull, and the Bastion maneuvers out of the bay almost as haphazardly as it did when it entered.`
+
+mission "Militia returning to Solace"
+	source "Solace"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Wandering around the main spaceport on Solace, you notice that Delta V is building a private spaceport near their main company offices. It seems that the work has been going on for a long while, but has only just been unveiled, because there is a crowd of interested spectators near the perimeter of the fence.`
```

Comment:
This paragraph confuses me. If it's just been unveiled, then why is it still being built?

```suggestion
			`Wandering around the main spaceport on Solace, you notice several rows of shiny new landing pads beside the Delta V company offices. A large banner announces that a you are witnessing the grand opening of Delta V's new private spaceport, and a crowd of interested spectators has gathered near the perimeter of the fence.`
```